.github/workflows/more-info-needed-closer.yml

```diff
@@ -0,0 +1,23 @@
+name: More Info Needed Closer
+on:
+  schedule:
+    - cron: 10 11 * * * # Run at 11:10 AM UTC (3:10 AM PST, 4:10 AM PDT)
+  workflow_dispatch:
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+      - name: Install Actions
+        run: cd ./.github/actions && npm install --production && cd ../..
+      - name: Stale Closer
+        uses: ./.github/actions/StaleCloser
+        with:
+          labels: more info needed
+          ignoredLabels: debugger
+          closeDays: 14
+          closeComment: "This issue has been closed automatically because it needs more information and has not had recent activity."
+          pingDays: 80
```

Comment:
I don't understand the 80 for ping vs 14 for close here.  Won't the issue be closed and locked already?

Reply:
StaleCloser.ts is fairly brief, so should be easy to decipher.  The issue is not closed if pingDays is set and if the last commenter is not specified in additionalTeam and does not have write or admin access to the repo (or is a bot).  If there is an assignee, a comment will be posted to ping them instead (once per pingDays).

Reply:
But we'd be pinging after the issue is closed?  80 > 60

Reply:
No, the issue would not be closed if the last comment was made by a user, and not us (as we may not have seen their comment, and the issue may be on our plate, not theirs).  Instead, after pingDays, the assignee is pinged.  After an issue is closed, no-one will be pinged.  The StaleCloser action only queries for issues that are: "is:open is:unlocked"...  Though, the code as written does appear to assume pingDays will be greater than closeDays, as the initial query is based on the closeDays value.

Reply:
Got it. So "closeDays" is for when it's assigned to us and "pingDays" is for when it's assigned to them.

Reply:
If you are using the term 'assigned' loosely, then yes.  If you are using it to refer to assigning the issue using the 'assignees' field, then no.  closeDays is for when we were the last ones to comment on the issue.  Otherwise, if a user was the last to comment, it doesn't get closed, and IF there is an assignee they are pinged.